consensus/polybft/fsm.go

```diff
@@ -400,9 +397,21 @@ func (f *fsm) VerifyStateTransactions(transactions []*types.Transaction) error {
 			if !verified {
 				return fmt.Errorf("invalid signature for tx = %v", tx.Hash)
 			}
+		case *contractsapi.CommitEpochFunction:
+			uptimeTransactionExists = true
```

Comment:
Should we validate that uptime transaction is the first transaction in the block?

Reply:
I don't think we need that, since there is a discussion that `commitEpoch` is not necessarily the first transaction in block.
I would add a check that I figured out right now, is that there can only be one `commitEpoch` transaction. That is something we are missing.

Reply:
Fix for two `commitEpoch` txs in block: https://github.com/0xPolygon/polygon-edge/pull/1186/commits/78df39a2c6c62985ced07d121da713eda5ec1245